src/sentry/integrations/pagerduty/notify_action.py

```diff
@@ -44,24 +46,22 @@ def clean(self):
                 raise forms.ValidationError(_("Invalid account"), code="invalid")
 
         service_id = cleaned_data.get("service")
-
-        service = PagerDutyService.objects.get(id=service_id)
-
-        # need to make sure that the service actually belongs to that integration - meaning
-        # that it belongs under the appropriate account in PagerDuty
-        if not service.organization_integration.integration_id == integration_id:
-            params = {
-                "account": dict(self.fields["account"].choices).get(integration_id),
-                "service": dict(self.fields["service"].choices).get(int(service_id)),
-            }
-
-            raise forms.ValidationError(
-                _(
-                    'The service "%(service)s" does not exist or has not been granted access in the %(account)s Pagerduty account.'
-                ),
-                code="invalid",
-                params=params,
-            )
+        if service_id:
+            service = PagerDutyService.objects.filter(id=service_id).first()
```

Comment:
Since you're using `.first()` does that mean there could be multiple `PagerDutyServices` for a single `service_id`? Is this throwing away others?

Reply:
In the `PagerDutyServices` table `id` is the primary key so it's guaranteed to be unique. I'm just using `first()` to get an option.

Reply:
So why not use `.get()`?

Reply:
I was trying to avoid another try/catch but you're right it's a better fit.